### DISCUSSION TITLE
[Github Actions] FastAPI People keep failing due to rate limiting

### DISCUSSION THREAD

**Discussion by ceb10n** [fastapi/fastapi#12543] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
n/a
```

### Description

This is not a problem related directly to FastAPI. 🙏

I was looking at the actions from the repository by curiosity, and I noticed that all runs from the FastAPI people keeps failing due to Github's rate limits, and of course it's because (thankfully) the project is growing and there are to many people helping with discussions, translations, sponsors, etc.:

```shell
ERROR:root:[{'type': 'RATE_LIMITED', 'message': 'API rate limit exceeded for user ID 1326112.'}]
ERROR:root:{"errors":[{"type":"RATE_LIMITED","message":"API rate limit exceeded for user ID 1326112."}]}
```

I think that there are several approaches to solve this problem:

- **Keep track in the repo**

I think that the easiest solution is to keep track of past runs in the repo, maybe with a json file or something like that.

- **Use multiple tokens**

Another way possible, is to use several gh tokens, and rotate them every time the rate limit is reached. I don't if it's even allowed by github, but it is a possibility.

- **Create a Github App**

This is my favorite solution, although it might involve some costs.

We can create a `Github App` and subscribe to related webhooks, like `discussion`, `discussion_comment`, `issues`, `issue_comment`, `pull_request`, etc.

This can be developed using (of course) `FastAPI`, `SQLModel`, etc. and serving as a showcase and a real example of the usage.

With all the data received from the webhooks, we can expose the metrics in an API, allowing an easy way to create pages like the FastAPI People.

And this would help other projects that I know is using an action inspired by the one created in this project, like  `Pydantic` 🚀.

I don't know if this problem is already addressed by the team, but I can help with it if you need 😃 

### Operating System

Linux

### Operating System Details

n/a

### FastAPI Version

n/a

### Pydantic Version

n/a

### Python Version

n/a

### Additional Context

n/a

**Comment by YuriiMotov** [comment]
Seems that it has been fixed.
